corehq/apps/campaign/models.py

```diff
@@ -1,4 +1,14 @@
 from django.db import models
+from django.db.models import Prefetch
+
+from corehq.apps.userreports.models import ReportConfiguration
+
+
+class DashboardManager(models.Manager):
+    def get_queryset(self):
```

Comment:
I wonder if we should reconsider this?

I assume the reports would be loaded asynchronously and the dashboard would be loaded as a part of the page load itself, so we won't need reports with dashboard?

Reply:
We could instead consider loading dashboard with report if we end up checking things on dashboard model when working with reports.